src/databricks/labs/remorph/config.py

```diff
@@ -11,14 +11,15 @@ class MorphConfig:
     source: str
     input_sql: str
     output_folder: str | None
-    sdk_config: Config | None
+    sdk_client: WorkspaceClient | None
     skip_validation: bool = False
+    serverless_warehouse_id: str = None
```

Comment:
if we use `DATABRICKS_WAREHOUSE_ID` env variable, then it can be in the `sdk_config` automatically

Reply:
i am explicitly blocking the addition of a new variable for what we already have equivalent for